### PR TITLE
Fix reply on code review

### LINKED ISSUE #9891: Reply Button Not Working (Pull Request - File Changed - Split View)
- Gitea version (or commit ref): 1.10.1. 
- Git version:
- Operating system: Ubuntu 16.04
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

Example URL: https://try.gitea.io/choiip/test_repo/pulls/3/files?style=split&whitespace=

## Description

I couldn't use the Reply button on Split View - File Changed - Pull Request section. 

When I click Reply button, it doesn't put information. I tried on version 1.10.1. and Test Instance.
...

## Screenshots
image

### PR DESCRIPTION
**Pull request by lunny**
Should fix #9891

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]
Just one question, anyway LGTM

File: web_src/js/index.js [hunk 1]
```diff
@@ -3547,6 +3547,14 @@ window.cancelCodeComment = function (btn) {
     form.closest('.comment-code-cloud').remove();
   }
 };
+
+window.submitReply = function (btn) {
+  const form = $(btn).closest('form');
+  if (form.length > 0 && form.hasClass('comment-form')) {
```
[the remark this answers is not in the document]
Comment (lunny) [reply web_src/js/index.js:3565]: Just copied from onCodeReviewClicked. :)

**Comment by silverwind** [comment]
Why not just use `<button type="submit">`. Inline HTML `onclick` should be avoided.

**Review by silverwind — CHANGES_REQUESTED** [review]
See above.

**Comment by lunny** [comment]
> Why not just use `<button type="submit">`? Inline HTML `onclick` should be avoided ([some](https://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice) [reasons](https://www.reddit.com/r/javascript/comments/3kamqn/why_is_inline_onclickfunction_considered_such_a/)) and if it has to be JS, use `addEventListener`, but I'm pretty certain this issue can be fixed without using any JS because it's a simple form submission that browers can do just fine without JS (unless you want to submit it async, but this case is sync submission).

I have tried that, but it's not work.

**Review by silverwind — COMMENTED** [review]
Guess I won't block as this is a bugfix, but these form submissions should be cleaned up to work without JS.

**Review by 6543 — APPROVED** [review]
yes a other pr can minimize js for souch things :)

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**